src/modules/crudData.js

```diff
@@ -0,0 +1,161 @@
+class Functionality {
+  static localGetItem() {
+    let data;
+    if (localStorage.getItem('todoData') === null) {
+      data = [];
+    } else {
+      // eslint-disable-next-line no-unused-vars
+      data = JSON.parse(localStorage.getItem('todoData'));
+    }
+    return data;
+  }
+
+  // add the data to local storage
+  addData = (newData) => {
+    this.newData = newData;
+    const data = Functionality.localGetItem();
+    data.push(newData);
+    localStorage.setItem('todoData', JSON.stringify(data));
+  };
+
+  // displays the to do list data
+  displayData = (newData) => {
+    this.newData = newData;
+    const todoList = document.querySelector('.task-list');
+    const todoTask = document.createElement('div');
+    todoTask.className = 'task';
+    todoTask.id = newData.index;
+    // todoTask.draggable = true;
+
```

Comment:
- [optional] It would be more professional if you can remove this commented line of code in order to clean your codebase. 👍